### PR TITLE
Use REST endpoint for retrieving applications in stream summary

### PR DESCRIPTION
**Pull request by ilayaperumalg**

### CODE REVIEW

**Comment by BoykoAlex** [comment]
We'd need to mirror this PR for the master branch as well

File: ui/src/app/streams/streams.service.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { StreamHistory } from './model/stream-history';
 import { catchError, map } from 'rxjs/operators';
 import { Platform } from '../shared/model/platform';
 import { DataflowEncoder } from '../shared/support/encoder.utils';
+import {AppRegistration} from "../shared/model";
```
Comment (jvalkeal) [inline ui/src/app/streams/streams.service.ts:16]: Format for linting

File: ui/src/app/streams/streams.service.ts [hunk 2]
```diff
@@ -350,4 +351,13 @@ export class StreamsService {
       );
   }
 
+  getApplications(name: string): Observable<AppRegistration[]> {
+    const headers = HttpUtils.getDefaultHttpHeaders();
+    return this.httpClient
+      .get<any>(`/streams/definitions/${name}/applications`, { headers })
```
Comment (jvalkeal) [inline ui/src/app/streams/streams.service.ts:357]: any should be AppRegistration[]

**Comment by jvalkeal** [comment]
Merged per 75ec61c3748c043ea2dfa725ae86d78c2ec61943

**Pull request closed without merging**